creator-cards/sean-killeen.json

```diff
@@ -1,33 +1,49 @@
-﻿{
+{
   "name": "Sean Killeen",
   "sectionTitle": null,
   "countryCode": "us",
   "slogan": "",
   "type": "CREATOR",
-  "bio": "",
-  "tags": ".NET, C#, Automated Testing, Azure, Terraform",
+  "bio": "A mind at work. | Director, Modern Software Delivery @ Excella | MS MVP | NUnit docs | .NET DC | Teacher. Trying to be all the human I can be with the time I have. (he/him)",
```

Comment:
You can change your BIO and all social links on the page directly in your settings, its just kept in json for reference we might get rid of sooner or later.

Reply:
Thats why you might spot differences between how we have the social part in json, some of that is old left-overs as since we added the option to added all that to the page its no longer used, we probably need to update our guide here a bit